lib/compiler/ops-to-jsir.js

```diff
@@ -268,15 +268,24 @@ module.exports = class OpCompiler {
         });
     }
 
-    _compileAtTimer(ast) {
+    _compileAtTimer(streamop) {
         let tryCatch = new JSIr.TryCatch("Failed to invoke at-timer");
         this._irBuilder.add(tryCatch);
         this._irBuilder.pushBlock(tryCatch.try);
 
         let iterator = this._irBuilder.allocRegister();
-        let time = compileValue(this._irBuilder, ast.time, this._currentScope);
+        let timeArray = this._irBuilder.allocRegister();
 
-        this._irBuilder.add(new JSIr.InvokeAtTimer(iterator, time));
+        this._irBuilder.add(new JSIr.CreateTuple(streamop.time.length, timeArray));
+        for (let i = 0; i < streamop.time.length; i++) {
+            let time = compileValue(this._irBuilder, streamop.time[i], this._varScope);
```

Comment:
this._varScope should probably be this._currentScope ? That's what the code before used...

Reply:
Fixed!